docs/notebooks/LRSinkhorn.ipynb

```diff
@@ -252,7 +254,7 @@
     "id": "BV7wO_Dcijc3"
    },
    "source": [
-    "We compute plans satisfy a rank constraint $r$, for various values of $r$,"
+    "We compute plans satisfy a rank constraint $r$, for various values of $r$."
```

Comment:
same, i know it wasn't you, but "that satisfy"

Reply:
Done.